### DISCUSSION TITLE
example_Deep Gaussian Process

### DISCUSSION THREAD

**Discussion by findoctorlin** [cornellius-gp/gpytorch#2227] [Q&A] [answered]
Hey guys!

I am running the code for Deep Gaussian Process. Can someone tell me why the predictive mean here is 10x3320 instead of 1x3320? Since the output dimension is 1.
[
<img width="766" alt="DGP" src="https://user-images.githubusercontent.com/57867183/207755393-338be648-9fb1-42c6-b6bc-66d60e4b2025.png">
](url)
Thank you!!

**Comment by gpleiss** [comment, marked as answer]
The deep GP model outputs 10 samples of the (approximate) predictive posterior distribution. (We draw samples from the hidden layer, and pass them into the final layer, which outputs a MVN. The output is `num_samples` MVN distributions.)
